### PR TITLE
make the round route button bounce on the first visit

### LINKED ISSUE #123: Add nudge for the route selector
(Based on feedback from Sava)

When the user opens the site for the first time, they see a map - but no routes, nor vehicles. Since nothing else happens, some might be confused about what to do next.

The UI needs a nudge that will hint them what to do to get started. Therefore:

- when the UI is loaded
- and there are no previously selected routes (e.g. from previous sessions)
- the nudge should be triggered

The nudge itself:
1. the route selection icon should bounce (a pulsating effect might also work, and other alternatives can be discussed)
2. its tooltip should be shown

### PR DESCRIPTION
**Pull request by dasshield**
Closes #123 

How it looks:
Roata Wăy (1)

### CODE REVIEW

File: src/app.component.tsx [hunk 1]
```diff
@@ -13,11 +11,21 @@ import classes from './app.module.scss'
 export function AppComponent() {
   const { t } = useTranslation()
   useDocumentTitle(t('label.title'))
-  const [isOpenRouteSelect, setIsOpenRouteSelect] = useState(false)
+  const [isOpenRouteSelect, setIsOpenRouteSelect] = useState<boolean>(false)
   // Maybe keep it in local storage?
   const [showUserLocation, setShowUserLocation] = useState<number | undefined>(undefined)
   const [selectedRoutes, setSelectedRoutes] = useSelectedRoutes()
-  const [openNote, setOpenNote] = useState(true)
+  const [firstVisit, setFirstVisit] = useState<boolean>(() => !areStoredRoutesEmpty())
```
Comment (dasshield) [inline src/app.component.tsx:18]: Thought about smth like this, but this way `Tooltip` switches between uncontrolled & controlled modes
```
<Tooltip
  title={t('label.pickRoute')}
  placement="left"
  {...firstVisit ? { open: true } : {}}
>...
```

**Review by iamandrewluca — APPROVED** [review]

File: src/app.component.tsx (hunk 1 above)
Comment (iamandrewluca) [reply src/app.component.tsx:18]: Let's keep it how is now.

Comment (iamandrewluca) [reply src/app.component.tsx:18]: Also in most cases, there is no need for a gif preview :) Netlify takes care of deploying a frontend instance, so everyone can test the feature.

Comment (dasshield) [reply src/app.component.tsx:18]: Just in case ;)

**Comment by iamandrewluca** [comment]
@dasshield Thanks for contribution.

**Pull request merged**